nss-day-cohort-41/react-nutshell-the-hazelnuts
# Sf post friend

# Description
Fixes # 7
1. Delete functionality fixed on friends list --previously was not repopulating the usernames properly, but now once "x" is clicked it should only delete that one username and repopulate every other username not deleted
2. Currently every message should be linked properly to session storage user so it will dynamically show edit button and add friend button will show on messages that are not the user
3. Friends list should only show friends of current active user and no one else's friends

## Type of change

# Testing Instructions
-Pull down sf-post-friend and serve json and npm start; then go to friends list and click "x" on friend you want to remove and it should be removed from friends list without affecting other usernames;

-edit and add friend buttons should be showing dynamically based on session storage Id

-check your database to make sure only your friends of the user signed in are showing

# Checklist:
